cloudfront/iiif.wellcomecollection.org/terraform/cf_behaviours.tf

```diff
@@ -100,7 +100,7 @@ locals {
     {
       path_pattern     = "image/*"
       target_origin_id = "dlcs_images"
-      headers          = []
+      headers          = ["Host"]
```

Comment:
When testing this the addition of the `Host` header forwarding seems to break the behaviour entirely!

Reply:
Odd - reverted to `[]`. I added `"Host"` thinking I'll need to see the original host in dlcs to update some returned values based on whether the request was made directly or via CF. I can test without `"Host"` and troubleshoot adding back in if need be.